pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue

```diff
@@ -32,14 +33,6 @@ export default defineComponent({
       default: null
     },
 
-    /**
-     * Random ID generated for binding label to input.
-     */
-    id: {
-      type:    String,
-      default: String(Math.random() * 1000)
-    },
-
```

Comment:
Removing the `id` prop does alter the contract of the component and alters where the `id` will be placed; with the removal the `id` is applied to the container div of the checkbox, while it was previously supplied as the name to the `input` element in this component.

Reply:
reverted